src/elements/content-sidebar/__tests__/__snapshots__/SidebarFileProperties.test.js.snap

```diff
@@ -23,6 +24,7 @@ exports[`elements/content-sidebar/SidebarFileProperties render() should render I
 exports[`elements/content-sidebar/SidebarFileProperties render() should render ItemProperties for anonymous uploaders 1`] = `
 <LoadingIndicatorWrapper>
   <ItemProperties
+    archivedAt={1726832355000}
```

Comment:
Ideally, all our endpoints should use the same format for dates. We can address that as a separate change, though, if needed.